cmd/podman/common/completion.go

```diff
@@ -964,6 +964,13 @@ func AutocompleteUserNamespace(cmd *cobra.Command, args []string, toComplete str
 	return results, directive
 }
 
+// AutocompleteBaseHostsFile - Autocomplete base hosts file options.
+// -> "image", "none", paths
+func AutocompleteBaseHostsFile(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	BaseHostsFileModes := []string{"image", "none"}
```

Comment:
I don't see a reason for image it is equivalent to "".  "none" conflicts with --no-hosts, if you want to replace --no-hosts with this and make --no-hosts hidden, I would be fine with that.